debian/files

```diff
@@ -0,0 +1,2 @@
+install-win31_0.1-1_all.deb contrib/otherosfs optional
```

Comment:
I think this is auto-generated.

Reply:
Indeed, I has missed that one! Thanks!!!